velox/functions/sparksql/registration/RegisterDatetime.cpp

```diff
@@ -110,6 +110,12 @@ void registerDatetimeFunctions(const std::string& prefix) {
       Varchar,
       int32_t,
       Timestamp>({prefix + "timestampadd"});
+  registerFunction<
+      TimestampAddFunction,
+      Timestamp,
+      Varchar,
+      int64_t,
+      Timestamp>({prefix + "timestampadd"});
```

Comment:
Would you please update the supported types for `value`?

https://github.com/facebookincubator/velox/blob/main/velox/docs/functions/spark/datetime.rst?plain=1#L288